mixer/adapter/kubernetesenv/kubernetesenv.go

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 //go:generate $GOPATH/src/istio.io/istio/bin/mixer_codegen.sh -f mixer/adapter/kubernetesenv/config/config.proto
```

Comment:
I wonder if 
```
//go:generate $GOPATH/src/istio.io/istio/bin/mixer_codegen.sh -a mixer/adapter/kubernetesenv
```
would be better? The script should check `-f config/config.proto` and `-t template/template.proto` ?

Reply:
I think having a convention for config + template for adapters is a decent idea. There are only two adapters currently in this situation currently in this codebase. If it is OK, let's put that in a follow-on PR.